README.md

```diff
@@ -1,2 +1,73 @@
-# tau
-TAU (Tizen Advanced UI) project for supporting web app development based on web-based ui widget components library.  This will be replacement and clone of previously existing https://review.tizen.org/gerrit/#/admin/projects/platform/framework/web/tau
+# TAU
+
+The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components. The components represent a visual UI element, such as a button or slider, which gives you interaction and manipulation features.
+
```

Comment:
I suggest adding a Getting Started section name according to the
https://gist.github.com/PurpleBooth/109311bb0361f32d87a2
I love this example as a  good Readme file

Reply:
I modified PR. I left several section marked as to do. File AUTHOR needs update. Project has two licenses files MIT & FLORA.